Makefile

```diff
@@ -0,0 +1,135 @@
+all: fmt check
+
+# Always keep the last released version here
+VERSION ?= 0.0.1
```

Comment:
What about getting version from the latest tag.

Reply:
I'd prefer to have this deterministic and explicit: if someone isn't up-do-date on his env, the result will look different than what is expected. Also, by having it explicit, one can easily rebuild a specific version.

Reply:
ok, make sense